gensim/models/word2vec.py

```diff
@@ -1563,7 +1563,7 @@ def __init__(self, source, max_sentence_length=MAX_WORDS_IN_BATCH, limit=None):
         """
         `source` should be a path to a directory (as a string) where all files can be opened by the
         LineSentence class. Each file will be read up to
-        `limit` lines (or no clipped if limit is None, the default).
+        `limit` lines (or not clipped if limit is None, the default).
```

Comment:
The docs are not clear -- does the "will process all files in a directory" work recursively?

Reply:
It does not. Maybe wishlist? I've clarified the docs.